api-test/run_k6_local.sh

```diff
@@ -9,5 +9,8 @@
 if [[ "$(pwd)" =~ .*"api-test".* ]]; then
     cd ..
 fi
-postman-to-k6  api-test/ApiConfig.postman_collection.json -e api-test/local.postman_environment.json -o ./k6-script.js
-k6 run --http-debug=full --vus 2 --duration 5s ./k6-script.js
+
+bash api-test/postman-to-k6-converter.sh api-test/local.postman_environment.json
```

Comment:
U could omit this parameter `api-test/local.postman_environment.json` and set it only in to AZ pipeline.
In this way it's clear that without param is local otherwise we use other config

Reply:
ok